core/src/main/java/org/apache/struts2/config/entities/ConstantConfig.java

```diff
@@ -257,6 +260,9 @@ private String classesToString(Set<Class<?>> classes) {
         map.put(StrutsConstants.STRUTS_EXCLUDED_CLASSES, classesToString(excludedClasses));
         map.put(StrutsConstants.STRUTS_EXCLUDED_PACKAGE_NAME_PATTERNS, StringUtils.join(excludedPackageNamePatterns, ','));
         map.put(StrutsConstants.STRUTS_EXCLUDED_PACKAGE_NAMES, StringUtils.join(excludedPackageNames, ','));
+        map.put(StrutsConstants.STRUTS_DEV_MODE_EXCLUDED_CLASSES, classesToString(devModeExcludedClasses));
```

Comment:
LGTM. 👍 

Your implementation gives developers the flexibility to change exclusions in a clear and explicit manner for `devMode` true.